tests/components/application-connector/resources/charts/gateway-test/templates/basic-cases.yml

```diff
@@ -30,6 +30,48 @@ spec:
           credentials:
             secretName: basic-test
             type: Basic
+    - displayName: csrf basic
+      name: csrf-basic
+      providerDisplayName: Basic with CSRF
+      description: Should return 200 for Basic Auth with CSRF optimistic scenario
+      id: "{{ uuidv4 }}"
+      entries:
+        - type: API
+          targetUrl: "http://test-app.{{ .Values.namespace  }}.svc.cluster.local:8080/v1/api/csrf-basic/ok"
+          centralGatewayUrl: "http://central-application-gateway.kyma-system:8080/test-app/csrf-basic"
+          credentials:
+            secretName: basic-test
+            type: Basic
+            csrfInfo:
+              tokenEndpointURL: "http://test-app.{{ .Values.namespace  }}.svc.cluster.local:8080/v1/api/csrf/token"
+    - displayName: bad csrf token basic
```

Comment:
In my opinion, separating positive and negative cases into separate Application CRD will increase the readability of the tests.

Reply:
Hmm, I'm not sure as we probably will have only one or two positive cases per authorization method (so also per an Application) and the cases are going to be quite well described with the return codes. If you still think we should separate it, I'll do it after the rebase onto main with the PR that adds the multiple Application map to the test suite, @mvshao